packages/viewer/src/components/viewer-pin-group/vertex-pin-group.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+  Component,
+  Fragment,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+import { Dimensions, Matrix4, Point, Vector3 } from '@vertexvis/geometry';
+import classNames from 'classnames';
+
+import { Viewport } from '../..';
+import { PinController } from '../../lib/pins/controller';
+import { isDefaultPin, isTextPin, Pin } from '../../lib/pins/entities';
+import { PinModel } from '../../lib/pins/model';
+import { translatePointToScreen } from '../viewer-markup/utils';
+import { getClosestCenterToPoint } from './utils';
+
+interface ComputedPoints {
+  pinPoint: Point.Point;
+  labelPoint?: Point.Point;
+}
+@Component({
+  tag: 'vertex-viewer-pin-group',
+  styleUrl: 'vertex-pin-group.css',
+  shadow: false,
```

Comment:
Any reason this can't be `shadow: true`?

Reply:
Yeah, I had chatted with @danschultz about this. He suggested leaving this as a false shadow dom in order for the dom-renderer to pick it up.